fail2ban/server/banmanager.py

```diff
@@ -26,6 +26,9 @@
 
 from threading import Lock
 
+import dns.exception
```

Comment:
Informative message in the output, like "Banned ASN list: Failed to import dnspython" ? or just in the log and have it print out an empty list?